### DISCUSSION TITLE
Make it work on hostinger

### DISCUSSION THREAD

**Discussion by clk66** [uvdesk/community-skeleton#742] [General]
Hi all,

does anyone have any experience making it work on hostinger? Any guide I can follow or any help?
Cannot get it to work properly.

Thanks

**Comment by nicokr01** [comment]
Hi, would you like to host UVdesk on a VPS server ? If yes, just download apache2, mysql, php and composer and then download the repo with composer and then start the installation wizard, you also have to create a SQL table

**Comment by alex23232-tech** [comment]
[HR Business Solutions](https://hrbs.com.pk/)! We are dedicated to simplifying your HR processes with our comprehensive services, including [recruitment](https://hrbs.com.pk/recruitment-headhunting/), payroll outsourcing, and employee training. Let's work together to enhance your business efficiency!" Feel free to modify it to better fit your style!
